### PR TITLE
Update Ukrainian.properties

### PR DESCRIPTION
**Pull request by Prosta4okua**

### CODE REVIEW

File: android/assets/jsons/translationsByLanguage/Ukrainian.properties [hunk 1]
```diff
@@ -134,7 +134,7 @@ Longhouse = Довгий будинок
 
 Forge = Кузня
  # Requires translation!
-+15% production of land units = 
++15% production of land units = +15% до Виробництва
```
Comment (JackRainy) [inline android/assets/jsons/translationsByLanguage/Ukrainian.properties]: Missing ` наземних підрозділів`

File: android/assets/jsons/translationsByLanguage/Ukrainian.properties [hunk 2]
```diff
@@ -824,15 +824,15 @@ Specialist Buildings = Спеціалізовані будівлі
...
 Specialists = Спеціалісти
  # Requires translation!
-[specialist] slots = 
+[specialist] slots = Слотів: [specialist]
  # Requires translation!
-Engineer specialist = 
+Engineer specialist = Інженер-спеціаліст
  # Requires translation!
-Merchant specialist = 
+Merchant specialist = Купец-спеціаліст
```
Comment (JackRainy) [inline android/assets/jsons/translationsByLanguage/Ukrainian.properties]: `Торговець` according to other translations, neither `Купець` nor `Купец` (not ukrainian word)

File: android/assets/jsons/translationsByLanguage/Ukrainian.properties [hunk 3]
```diff
@@ -824,15 +824,15 @@ Specialist Buildings = Спеціалізовані будівлі
...
  # Requires translation!
-Scientist specialist = 
+Scientist specialist = Учений-спеціаліст
```
Comment (JackRainy) [inline android/assets/jsons/translationsByLanguage/Ukrainian.properties]: `Спеціаліст-науковець` according to other translations

File: android/assets/jsons/translationsByLanguage/Ukrainian.properties [hunk 4]
```diff
@@ -994,11 +994,11 @@ Resolution = Роздільність
...
 Map editor = Редактор мап
  # Requires translation!
-Create = 
+Create = Створити
  # Requires translation!
-New map = 
+New map = Нова мапа
  # Requires translation!
-Empty = 
+Empty = Пусто
```
Comment (JackRainy) [inline android/assets/jsons/translationsByLanguage/Ukrainian.properties]: `Пусто` (not ukrainian word) - `Порожньо`

**Comment by yairm210** [comment]
I'll wait with this until you 2 resolve it to your satissfaction =)

**Comment by Prosta4okua** [comment]
@JackRainy

**Comment by yairm210** [comment]
Ahhh sorry I accidentally did something github-y that closed it it a strange way that it can't be opened =/
Could you please submit the pull request again? Thanks 
Sorry again =(

**Comment by JackRainy** [comment]
Looks good, @Prosta4okua !
Please, resubmit the PR.

**Pull request closed without merging**